systems-management/nisysmgmt.yml

```diff
@@ -587,7 +587,25 @@ definitions:
         example: 19.0
     required: [activated]
 
-  ManagedSystemDataProperty:
+  ManagedSystemDataPropertyList:
+    description: Property of a system composed by the last updated timestamp and a list with the actual data.
+    type: object
+    properties:
+      lastUpdatedTimestamp:
+        description: Last updated timestamp.
+        type: string
+        format: iso-date-time
+        example: '2019-02-11T13:32:06.030958Z'
+      data:
+        description: The actual data.
+        type: array
+        items:
+          type: object
+          example: 
+            - Lab1
```

Comment:
Did you intend to make the example an array inside the array? As in, this example is for an item of the array, not the data array itself.

Reply:
Oops.. it was a copy-paste wrong indent problem for the example. However this definition is for SysAPI data, which is a list of property-bags (and we don't really care to define the property-bags, hence we let them 'objects'). 
I removed the example altogether since we don't really have a definition for the property-bags.
I see most of them have a resource_uri and an array of properties which have type/tag/value. However, our service doesn't consume any of those, so we take whatever sysAPI is publishing and return it as 'object', that way we won't have to change when sysAPI changes.